### DISCUSSION TITLE
Trying to figure out path-based routing for Emby

### DISCUSSION THREAD

**Discussion by pentago** [nginx-proxy/nginx-proxy#1871] [Q&A]
My Emby instance is available at `http://192.168.1.11:8096/web/index.html` URL and I have a goal of accessing it via `http://192.168.1.11/emby`

I tried adding the following env vars to my docker-compose for Emby but I still can't access the desired URL (using `jwilder/nginx-proxy:dev-alpine`) and 503 gets returned:
```
- VIRTUAL_HOST=emby
- VIRTUAL_PORT=8096
- VIRTUAL_PATH=/emby
- VIRTUAL_DEST=/web
```

I read the docs to the best of my ability and likely missing something and would appreciate any guidance that helps to achieve what I'm after!

**Comment by buchdag** [comment]
Hi

```yaml
- VIRTUAL_HOST=emby
- VIRTUAL_PATH=/emby
```

Means nginx-proxy will proxy incoming request to `http://emby/emby` to your Emby container.

| `http://` | `emby`  | `/emby`  |
| ------------- | ------------- | ------------- |
|  | `VIRTUAL_HOST`  | `VIRTUAL_PATH`  |

Request to an IP won't be routed to your container with your configuration. You can set `VIRTUAL_HOST` to `192.168.1.11` if you want it to work this way, but I consider it to be somewhat on the hack-ish side. nginx-proxy should rather be used with proper hostnames.
